oneflow/core/functional/impl/common.cpp

```diff
@@ -65,27 +65,27 @@ Maybe<void> CheckInplaceCastValid(const std::shared_ptr<Tensor>& x,
   return Maybe<void>::Ok();
 }
 
-bool IsShapeCanExpandTo(const Shape& shape, const Shape& expand_shape) {
-  if (shape == expand_shape) { return true; }
-  if (expand_shape.NumAxes() < shape.NumAxes()) { return false; }
+Maybe<void> CheckShapeCanExpandTo(const Shape& shape, const Shape& expand_shape) {
+  if (shape == expand_shape) { return Maybe<void>::Ok(); }
+
+  CHECK_OR_RETURN(expand_shape.NumAxes() >= shape.NumAxes())
+      << Error::RuntimeError() << "Can not expand shape " << shape.ToString() << " to "
+      << expand_shape.ToString();
```

Comment:
<< Error::RuntimeError() << "Can not expand origin shape " << shape.ToString() << " to "
      << expand_shape.ToString() << " in  an inplace operation.";